priv/templates/extended_bin

```diff
@@ -328,10 +328,14 @@ case "$1" in
         HEART_COMMAND="$RELEASE_ROOT_DIR/bin/$REL_NAME $CMD"
         export HEART_COMMAND
 
-        mkdir -p "$PIPE_DIR"
+        if mkdir -p "$PIPE_DIR" ; then
```

Comment:
@lrascao  see the code result below.
```
#!/bin/sh

# this might be ok
if mkdir -p ~/tmp/some_dir ; then
    echo ok
else
    echo fail
fi



# this might be fail
if mkdir -p ~/root/test ; then
    echo ok
else
    echo fail
fi
```